EIPS/eip-5553.md

```diff
@@ -0,0 +1,260 @@
+---
+eip: 5553
+title: Works & Royalty Interest Representation
+description: A way of representing intellectual IP works and their respective royalty interests on chain
+author: Roy Osherove (@royosherove)
+discussions-to: https://ethereum-magicians.org/t/eip-821-simple-works-royalty-interest-representation-standard/10551
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-08-17
+requires: 20, 721
+---
+
+## Abstract
```

Comment:
[Editorial] It seems there are 3 subsections of Abstract. I wonder if it would be too long for what would be considered as "Abstract". Could you shorten the Abstract and then maybe everything else could be merge into Motivation or Specification?

Reply:
It's now shorter.